jwql/utils/maths.py

```diff
@@ -0,0 +1,154 @@
+"""Various math-related functions used by the ``jwql`` instrument monitors.
+
+Authors
+-------
+
+    - Bryan Hilbert
+
+Use
+---
+
+    This module can be imported as such:
+
+    >>> import maths
```

Comment:
@bhilbert4 I think this should be replaced with

```python
from jwql.utils import maths
```

Reply:
Fixed